archive/t/typescript/roman-numeral.ts

```diff
@@ -0,0 +1,77 @@
+// Roman Numeral Conversion
+
+declare const process: { argv: string[] };
```

Comment:
I think this needs to be removed. I took a look at other Typescript samples, and this is not present. I think it is causing [this error](https://github.com/TheRenegadeCoder/sample-programs/actions/runs/6393475716/job/17359498270?pr=3407#step:6:7698)

Reply:
stripped away!